### ISSUE TITLE
Document HAS

### ISSUE THREAD

**Issue by JJ** [perl6/doc#2965]
## The problem

It's not even mentioned. Seen in [this StackOverflow answer](https://stackoverflow.com/a/57616443/891440)

## Suggestions

Investigate it and add something in the NativeCall page or independently.

**Comment by jonathanstowe** [comment]
Yeah I keep forgetting about it, think I can't implement something and put it aside, then remember again when I don't have time.

**Comment by JJ** [comment]
Well, you don't need to do it yourself... We'll just leave it there for
now.

**Comment by Altai-man** [comment]
A source of inspiration from another ticket: https://stackoverflow.com/questions/50723156/how-to-declare-native-array-of-fixed-size-in-perl-6/50745483#50745483
